rpc/documentation/clientusage.md

```diff
@@ -60,14 +63,32 @@ import (
 	"github.com/decred/dcrd/dcrutil"
 )
 
-var certificateFile = filepath.Join(dcrutil.AppDataDir("dcrwallet", false), "rpc.cert")
+var (
+	certificateFile      = filepath.Join(dcrutil.AppDataDir("dcrwallet", false), "rpc.cert")
+	walletClientCertFile = filepath.Join(dcrutil.AppDataDir("dcrwallet", false), "client.pem")
+	walletClientKeyFile  = filepath.Join(dcrutil.AppDataDir("dcrwallet", false), "client-key.pem")
+)
 
 func main() {
-	creds, err := credentials.NewClientTLSFromFile(certificateFile, "localhost")
+	serverCAs := x509.NewCertPool()
+	serverCert, err := ioutil.ReadFile(certificateFile)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+	if !serverCAs.AppendCertsFromPEM(serverCert) {
+		fmt.Println(err)
```

Comment:
this should print some informative message instead of the err variable, which will always be nil here

Reply:
Resolved with https://github.com/decred/dcrwallet/pull/1907/commits/dcda70caa46a99108b838b8cab598265bec60814